.github/workflows/compatibility_tests.yml

```diff
@@ -23,85 +31,76 @@ jobs:
       - name: Build docker image
         run: docker build .
 
-  cellxgene-main-with-python-and-anndata-versions:
```

Comment:
this file is more easily reviewed by _not_ looking at the diff; it's basically a rewrite